documentation/docs/getting-started/installation.md

```diff
@@ -317,9 +317,9 @@ You can change your LLM provider and/or model or update your API key at any time
 
 <Tabs groupId="interface">
   <TabItem value="ui" label="goose Desktop" default>
-    1. Click the <PanelLeft className="inline" size={16} /> button in the top-left to open the sidebar.
-    2. Click the `Settings` button on the sidebar.
-    3. Click the `Models` tab.
+    1. Click the sidebar toggle button (☰) in the top-left corner to open the navigation menu
+    2. Click the `Settings` tab on the sidebar
+    3. In the Settings tab, click on `Model Catalog` under the `AI` section
```

Comment:
hii christie! thanks for your contribution. I'm curious if this line is actually true. I dont see this in my a Model Catalog section or an AI section in my app